app/models/subscriber.rb

```diff
@@ -0,0 +1,32 @@
+class Subscriber
+  def self.for_discussion(discussion)
+    new(discussion)
+  end
+
+  def initialize(discussion)
+    @discussion = discussion
+    @subscriptions = Subscription.where(discussion_id: discussion.id)
+  end
```

Comment:
any reason for having a `self.for_discussion` instead of just making this a named argument? `def initialize(discussion: discussion). I guess the end result is the same, but the current way allows you to still call the `new` method manually, which is probably not what you want